packages/dataviews/src/view-table.tsx

```diff
@@ -223,7 +223,7 @@ const _HeaderMenu = forwardRef( function HeaderMenu< Item extends AnyItem >(
 							onHide( field );
 							onChangeView( {
 								...view,
-								hiddenFields: view.hiddenFields.concat(
+								hiddenFields: view.hiddenFields?.concat(
```

Comment:
This feels wrong? Shouldn't it be `(view.hiddenFields ?? []).concat` ?

Reply:
good catch! 03ef404f206c274e6ea4e36b2aa7808e68963820